descwl_coadd/coadd.py

```diff
@@ -415,7 +418,7 @@ def make_coadd_old(
 
         if maskfrac > 0:
             # images modified internally
-            interp_nocheck(exp=exp, noise_exp=noise_exp, bad_msk=bad_msk)
+            _interp_nocheck_old(exp=exp, noise_exp=noise_exp, bad_msk=bad_msk)
```

Comment:
What's the reason for this not using `interpolator` that is passed in to the caller?

Reply:
This is the old code `make_coadd_old`, kept around as a reference